developing-system-programs.tex

```diff
@@ -45,7 +45,9 @@ \section{Flash Menu}
 
 That command would load the file {\tt newflashmenu.prg} at memory location \$50000.
 
-After that, you can simply press the reset button the side of the MEGA65i while holding down the NO SCROLL key, and it will boot again, and because it never left Hypervisor Mode during the previous boot cycle, it will run your updated flash menu program.
+After that, you can simply press the reset button the side of the MEGA65 while holding \specialkey{NO SCROLL} down,
```

Comment:
Saw a missing 'on' here:

... press the reset button **on** the side of the MEGA65...

Reply:
Oh, as this is the last thing on my radar, maybe I can try make the edit via github here and then approve the merge, let's see :)